Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -34,4 +34,23 @@ else
      Write-Warning "Could not find the summary report file $summaryMdPath"
 }
 
+. .\CodeAnalysisFilePathComputation.ps1
+$prcaEnabled = Get-TaskVariable -Context $distributedTaskContext -Name "GitPullRequestCodeAnalysisEnabled"
+if ($prcaEnabled -ieq "true")
+{
+    ComputeCodeAnalysisFilePaths $agentBuildDirectory
+
+    $sonarReportFolderPath = [System.IO.Path]::Combine($agentBuildDirectory, ".sonarqube", "out", ".sonar")
```

Comment:
This is the second time you compute this path so you may want to centralize it somewhere.